internal/testutil/base_service_suite.go

```diff
@@ -190,6 +192,7 @@ func (s *BaseServiceTestSuite) setupStores() {
 		ConnectionRepo:               NewInMemoryConnectionStore(),
 		EntityIntegrationMappingRepo: NewInMemoryEntityIntegrationMappingStore(),
 		SettingsRepo:                 NewInMemorySettingsStore(),
+		AlertLogsRepo:                NewMockAlertLogsRepo(),
```

Comment:
later: move this to InMemoryAlertLogsRepo and implement this as other inmemory classes